doc/sphinx/azhelpgen/doc_source_map.json

```diff
@@ -11,9 +11,11 @@
     "appservice": "src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py",
     "batch": "src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_help.py",
     "cdn": "src/command_modules/azure-cli-cdn/azure/cli/command_modules/cdn/_help.py",
+    "billing": "src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/_help.py",
     "cloud": "src/command_modules/azure-cli-cloud/azure/cli/command_modules/cloud/_help.py",
     "component": "src/command_modules/azure-cli-component/azure/cli/command_modules/component/_help.py",
     "cognitiveservices": "src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/_help.py",
+    "consumption": "src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py",
```

Comment:
I realize that this list is not already fully ordered but it'd help readability.